react-client/src/Components/DomInfoDialogBox/index.js

```diff
@@ -189,7 +194,7 @@ const DomInfoDialogBox = ({ id, idx, tag, classNames, parent, children, top, lef
             </div>
           )}
 
-          <ChildrenDetails children={domInfo.children} />
+          <ChildrenDetails children={domUtils.customChildFilter(domInfo.children)} />
```

Comment:
@hnbmunoz Just a question. Why did we filter the children and exclude children with `.custom-css` class?